resources/providers/PingIdentity/Provider.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace SocialiteProviders\PingIdentity;
+
+use GuzzleHttp\RequestOptions;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
+{
+    public const IDENTIFIER = 'PINGIDENTITY';
+
+    /**
+     * The scopes being requested.
+     *
+     * @var array
+     */
+    protected $scopes = [
+        'openid',
+        'profile',
+        'email'
+    ];
+
+    /**
+     * Get the authentication URL for the provider.
+     *
+     * @param  string  $state
+     * @return string
+     */
+    protected function getAuthUrl($state)
```

Comment:
Ah, perhaps I was unclear in my previous review.  Even though PHPDoc comments satisfy PHPStan, PHPStan is currently configured to not trust them since we have many incorrect type annotations.  Instead, it would be better to specify native PHP type annotations if possible.  In cases where the PHPDoc comment can be made more specific than native annotations (array shapes, for example), it's best to put the most specific native annotation, as well as a more specific PHPDoc comment.

For example:
```suggestion
     */
    protected function getAuthUrl(string $state): string
```